src/mocks/queries/linkedEvents/eventList.ts

```diff
@@ -0,0 +1,263 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { graphql } from 'msw';
+
+import tapahtumatEventListFrontPageCarousel from '../../responses/linkedEvents/eventList/tapahtumat-eventlist-frontpage.json';
+
+export const frontPageCarouselVariables = {
```

Comment:
How about adding some types?
```typescript

export type FrontPageCarouselVariables = {
  keyword: string;
  location: string;
  division: string;
  sort: string;
  superEventType: string;
  start: string;
  pageSize: number;
  include: string[];
};

export const frontPageCarouselVariables: FrontPageCarouselVariables = {
```
...

And then dropping the as unknown as Record<string, string> in the other place.
But that brings up the underlying issue that was hidden by the "as unknown as Record<string, string>"
use i.e. the pageSize being a number and URLSearchParams not accepting that as input:

`TS2345: Argument of type  FrontPageCarouselVariables  is not assignable to parameter of type`
```
string | Record<string, string> | string[][] | URLSearchParams
```
- Type  FrontPageCarouselVariables  is not assignable to type  Record<string, string> 
  - Property  pageSize  is incompatible with index signature.
    - Type  number  is not assignable to type  string 

So... hmm... is the pageSize being a number acceptable or should it be converted to string or filtered away before giving it to URLSearchParams? How do you see this?

Reply:
If I convert pageSize to string e.g. by:
```typescript
      const variables = {
        ...frontPageCarouselVariables,
        start: dateInPast,
        pageSize: `${frontPageCarouselVariables.pageSize}`,
      };
      const eventSearchUrl = new URLSearchParams(variables).toString();
```
then the next error in URLSearchParams is about the `include` parameter:
```
Property  include  is incompatible with index signature.
Type  string[]  is not assignable to type  string
```

Reply:
Anyway, this brings up the question whether the variables passed to URLSearchParams are being handled correctly?

Reply:
It's only for mocking and it can really be anything. These were just the parameters that were used in 1 of the examples. I haven't chosen the parameters -- a content manager has. So, I don't want to type it, because it can be just anything. These were the actual parameters for the response that was added there. I think this can be thought as a recorded request.

Reply:
I changed the init to URLSearchParams.